metadata.yaml

```diff
@@ -13,8 +13,10 @@ description: |
   machines running on your infrastructure.
 maintainers:
   - Canonical Commercial Systems Team <jaas-crew@lists.canonical.com>
-website: https://ubuntu.com/security/livepatch
-docs: https://discourse.ubuntu.com/t/ubuntu-livepatch-service/22723
+website:
+  - https://ubuntu.com/security/livepatch
+  - https://discourse.ubuntu.com/t/ubuntu-livepatch-service/22723
```

Comment:
This URL changes to https://discourse.ubuntu.com/t/livepatch-documentation/22723
Maybe we could change it to that or a shortened version like the one on line 19: https://discourse.ubuntu.com/t/22723

Reply:
This is actually the Discoure's behaviour. You can put anything there and it will redirect to the correct/fixed URL. By the way, this is from the past (and also is the same on the K8s charm repo). I think let's not touch it here.